models.py

```diff
@@ -0,0 +1,75 @@
+from django.db import models
+
+# Create your models here.
+class User(models.Model):	
+	name = models.CharField(max_length=50, unique=True)
+	emailid = models.EmailField()
```

Comment:
Can this be changed to email_id. Similar casing for contact_num(ber) in other places too.